custom_components/tuya_local/devices/tv_backlights.yaml

```diff
@@ -0,0 +1,223 @@
+name: TV Sync Backlights
+products:
+  - id: gaobbrxqiblcng2p
+    name: Loginovo Ambient TV Sync Light
+primary_entity:
+  entity: light
+  name: Backlight
+  icon: mdi:television-shimmer
+  dps:
+    - id: 102
+      type: boolean
+      name: switch
+    - id: 104
+      name: brightness
+      type: integer
+      range:
+        min: 0
+        max: 100
+      mapping:
+        - step: 1
+          scale: 0.392
+          constraint: video_mode
+          conditions:
+            - dps_val: ambi_mode
+              value_redirect: scene_brightness
+            - dps_val: color_mode
+              value_redirect: color_brightness
+    - id: 114
+      hidden: true
+      name: scene_brightness
+      type: integer
+      range:
+        min: 0
+        max: 100
+      mapping:
+        - step: 1
+          scale: 0.392
+    - id: 112
+      hidden: true
+      name: color_brightness
+      type: integer
+      range:
+        min: 0
+        max: 100
+      mapping:
+        - step: 1
+          scale: 0.392
+    - id: 111
+      name: rgbhsv
+      type: hex
+      format:
+        - name: r
+          bytes: 1
+        - name: g
+          bytes: 1
+        - name: b
+          bytes: 1
+    - id: 101
+      name: video_mode
+      type: string
+      hidden: true
+    - id: 101
+      name: effect
+      type: string
+      mapping:
+        - dps_val: video_mode
+          value: Video
+        - dps_val: music_mode
+          value: Music
+        - dps_val: film_mode
+          value: Wide film
+        - dps_val: color_mode
+          value: Static color
+        - dps_val: ambi_mode
+          value: Ambient scene
+secondary_entities:
+  - entity: number
+    name: Backlight intensity
+    type: integer
+    mode: slider
+    icon: mdi:speedometer-medium
+    dps:
+      - id: 103
+        name: value
+        type: integer
+        range:
+          min: 1
+          max: 3
+        mapping:
+          - step: 1
+            scale: 1
+  - entity: select
+    icon: mdi:shimmer
+    name: Scene select
+    format:
+      - name: scene_num
+        bytes: 2
+        range:
+          min: 1
+          max: 12
+    dps:
+      - id: 113
+        name: option
+        type: integer
```

Comment:
In tuya, numeric values for selecting from a fixed list are often actually strings surrounded by quotes.  Although the integration is forgiving about this it is better to match the protocol exactly in case of future changes that require more strictness.

Reply:
I am reading this data using `tinytuya.Device`, and it seems the data is returned as an integer

```json
{
	"dps": {
		"101": "color_mode",
		"102": true,
		"103": 1,
		"104": 100,
		"105": 0,
		"108": 64,
		"111": "ff2806",
		"112": 100,
		"113": 1,
		"114": 100,
		"115": false,
		"116": 12,
		"117": 7,
		"118": 150,
		"119": 150,
		"120": 0,
		"122": "ffffff",
		"125": "BRG",
		"126": false,
		"127": "RD200 _HDMI2.0_230318           V08.08.08.3840*2160p 60Hz 02. 02.296.M.SDR1921.1079.0. 38."
	}
}
```